scripts/pylib/twister/twisterlib.py

```diff
@@ -968,12 +969,10 @@ def handle(self):
         if harness.is_pytest:
             harness.pytest_run(self.log)
 
-        # sometimes a test instance hasn't been executed successfully with an
-        # empty dictionary results, in order to include it into final report,
+        # sometimes a test instance hasn't been executed successfully with no
+        # status, in order to include it into final report,
         # so fill the results as blocked
-
-        for case in self.instance.testcases:
-            case.status = 'blocked'
+        self.add_missing_testscases(harness)
```

Comment:
'add_missing_testscases' function has been updated, now the argument should be a reason string.
the same error in line 962 'self.add_missing_testscases(harness)'.